integration/nwo/fabric/packager/golang/platform.go

```diff
@@ -348,8 +348,15 @@ func moduleInfo(path string) (*ModuleInfo, error) {
 	}
 
 	moduleRootDir := filepath.Dir(string(moduleRoot))
+	// TODO: should remove all this hardcoded strings to be more generic, also see
+	// integration/nwo/fabric/topology/topology.go(AddNamespace) for another hardcoded path
 	index := strings.Index(moduleRootDir, "github.")
-	path = moduleRootDir[:index] + path
+	if index != -1 {
```

Comment:
what are we trying to do here and when is the -1 case applicable? when is it not?

Reply:
the -1 case applies when FSC is not installed in a path that contains "github." meaning that they are possibly not using the GOPATH convention. No?